packages/core/types/src/modules-sdk/index.ts

```diff
@@ -137,6 +137,7 @@ export type ModuleJoinerConfig = Omit<
    */
   schema?: string
   relationships?: ModuleJoinerRelationship[]
+  extraFields?: string[]
```

Comment:
suggestion: `extraDataFields`
at a first glance I thought this was the same that is inside `databaseConfig`.
And if you don't mind, we could add a comment below saying that `databaseConfig` won't be present on the public `__joinerConfig`, and the extra fields from there will be populated here.

Reply:
Ah wait, I think I added this in the wrong place. Should be in the loaded module type and it's a generated read-only attribute derived from the database config.

Does this need to be a configurable field?

Reply:
The use cases I see to keep it here, is if someone wants extra fields added to the database, but don't want to update them from the application.
Or if you replace the link module by a custom module or an external module and you can't infer these fields.
So I think it is a good thing to keep it configurable.

Reply:
Added 👍🏻